src/pages/semaphore-grants/index.tsx

```diff
@@ -0,0 +1,485 @@
+              <Stack mb={8} mt={{ base: 10, md: 0 }}>
+                <section id='description' ref={ref}>
+                  <PageSubheading mb={8}>Semaphore Community Grants</PageSubheading>
+
+                  <PageText mb={6}>
+                    The{' '}
+                    <Link
+                      fontWeight={700}
+                      color='brand.orange.100'
+                      href='https://appliedzkp.org/'
+                      isExternal
+                      _hover={{ textDecoration: 'none' }}
+                    >
+                      Privacy and Scaling Explorations Team
+                    </Link>{' '}
+                    is sponsoring a dedicated round of grants for applications that integrate the{' '}
+                    <Link
+                      fontWeight={700}
+                      color='brand.orange.100'
+                      href='https://semaphore.appliedzkp.org/'
+                      isExternal
+                      _hover={{ textDecoration: 'none' }}
+                    >
+                      Semaphore
+                    </Link>{' '}
+                    anonymous signaling protocol. Semaphore provides zero-knowledge primitives that
+                    enable developers to build powerful privacy preserving applications. With this
+                    grant round, we’re encouraging builders to try out these tools in real-world
+                    applications that matter to your communities.
+                  </PageText>
+                </section>
+              </Stack>
+
+              <Stack spacing={10}>
+                <section id='privacy-matters' ref={ref2}>
+                  <PageSection mb={6}>Privacy matters</PageSection>
+
+                  <PageText mb={6}>
+                    Whether it’s talking confidentially with friends, anonymously broadcasting
+                    messages, or simply wanting to choose which aspects of our identity we reveal in
+                    which contexts, privacy allows us to express ourselves freely and without fear.
+                    Privacy is a tool that should be accessible to as many people as possible, and
+                    PSE’s goal is to foster a collaborative community to make access to privacy a
+                    reality in everyday life.
+                  </PageText>
+
+                  <PageText mb={6}>
+                    If you’re a developer with an idea to use Semaphore for good, we want to hear
+                    from you!
+                  </PageText>
+                </section>
+              </Stack>
+
+              <Stack>
+                <section id='semaphore-as-privacy-layer' ref={ref3}>
+                  <PageSection mb={6}>Semaphore as a privacy layer</PageSection>
+
+                  <PageText mb={6}>
+                    Semaphore is designed to be a simple and generic privacy layer for decentralized
+                    applications (dApps) on Ethereum.{' '}
```

Comment:
```suggestion
                    applications (dapps) on Ethereum.{' '}
```

Based on the style guide we use on ethereum.org I think we should change this, but dont think this is a blocker.